docs/connectors/taps/postgres.rst

```diff
@@ -160,6 +160,11 @@ Example YAML for ``tap-postgres``:
     stream_buffer_size: 0                  # In-memory buffer size (MB) between taps and targets for asynchronous data pipes
     #batch_wait_limit_seconds: 3600        # Optional: Maximum time to wait for `batch_size_rows`. Available only for snowflake target.
 
+    # Options only for Fastsycn for Snowflake
```

Comment:
Typo in "Fastsycn".

Reply:
fixed by https://github.com/transferwise/pipelinewise/pull/696/commits/7680704de2e3a2c21df018847401726e60dda9cd